src/shared/lib/svgToPdfDownload.ts

```diff
@@ -13,49 +12,65 @@ function base64ToArrayBuffer(base64:string) {
     return bytes;
 }
 
-export default function (filename:string, svg:Element, servletUrl?: string) {
-    const req = svgToPdfRequest(svg, servletUrl);
+export default function svgToPdfDownload(fileName: string, svg: any) {
+    const width = svg.scrollWidth || parseInt((svg.attributes.getNamedItem('width') as Attr).nodeValue!), height = svg.scrollHeight || parseInt((svg.attributes.getNamedItem('height') as Attr).nodeValue!);
 
-    if (!req) {
-       return false;
-    }
-
-    req.end((err, res)=>{
-        if (!err && res.ok) {
-            fileDownload(base64ToArrayBuffer(res.text), filename);
+    // dealing with oncoprint svg (temporarily)
```

Comment:
this is temporary b/c @adamabeshouse will fix in actual library?

Reply:
Yes, correct. when we have the new oncoprint.js, these lines can be removed from here.